src/components/patch-details/patch-details.test.tsx

```diff
@@ -99,6 +142,7 @@ describe("Patch Details", () => {
         assetPk={assetWithPatches.id}
         assetPatch={mockAssetPatch}
         assetArea={mockAssetArea}
+        versionNumber={assetWithPatches.versionNumber}
```

Comment:
I think I saw in the asset sidebar component that versionNumber is expected to be nullable